io.github.dweymouth.supersonic.yml

```diff
@@ -0,0 +1,221 @@
+      env:
+    config-opts:
+      - -Dbuild-date=false
+      - -Dlibmpv=true
+      - -Dmanpage-build=disabled
+      - -Dalsa=enabled
+      - -Dcdda=disabled
+      - -Ddvbin=disabled
+      - -Ddvdnav=disabled
+      - -Djavascript=disabled
+      - -Dlibarchive=disabled
+      - -Dlua=disabled
+      - -Dpulse=enabled
+      - -Dsdl2=disabled
+      - -Dshaderc=disabled
+      - -Dvulkan=disabled
+    cleanup:
+      - /include
+      - /lib/pkgconfig
+    sources:
+      - type: archive
+        url: https://github.com/mpv-player/mpv/archive/refs/tags/v0.35.1.tar.gz
+        sha256: 41df981b7b84e33a2ef4478aaf81d6f4f5c8b9cd2c0d337ac142fc20b387d1a9
+        x-checker-data:
+          type: anitya
+          project-id: 5348
+          stable-only: true
+          url-template: https://github.com/mpv-player/mpv/archive/refs/tags/v$version.tar.gz
+    modules:
+    - name: libass
+      config-opts:
+        - --disable-static
+      sources:
+        - type: archive
+          url: https://github.com/libass/libass/releases/download/0.17.1/libass-0.17.1.tar.xz
+          sha256: f0da0bbfba476c16ae3e1cfd862256d30915911f7abaa1b16ce62ee653192784
+          x-checker-data:
+            type: anitya
+            project-id: 1560
+            stable-only: true
+            url-template: https://github.com/libass/libass/releases/download/$version/libass-$version.tar.xz
+        - type: script
+          commands:
+            - autoreconf -fiv
+          dest-filename: autogen.sh
+      cleanup:
+        - /include
+        - /lib/pkgconfig
+        - /share
+    - name: ffmpeg
+      cleanup:
+        - /include
+        - /lib/pkgconfig
+        - /share/ffmpeg/examples
+      config-opts:
+        # this is almost like io.mpv.Mpv, except we removed:
+        # --enable-libbs2b
+        # --enable-libjxl
+        # --enable-libmysofa
+        # --enable-libx264
+        # --enable-libx265
+        # because those would require yet another dependency tree
+        - --disable-debug
+        - --disable-doc
+        - --disable-programs
+        - --disable-static
+        - --enable-encoder=png
+        - --enable-gnutls
+        - --enable-gpl
+        - --enable-shared
+        - --enable-version3
+        - --enable-libaom
+        - --enable-libass
+        - --enable-libdav1d
+        - --enable-libfreetype
+        - --enable-libmp3lame
+        - --enable-libopus
+        - --enable-libtheora
+        - --enable-libv4l2
+        - --enable-libvorbis
+        - --enable-libvpx
+      sources:
+        - type: git
+          url: https://github.com/FFmpeg/FFmpeg.git
+          mirror-urls:
+            - https://git.ffmpeg.org/ffmpeg.git
+          commit: ea3d24bbe3c58b171e55fe2151fc7ffaca3ab3d2
+          tag: n6.0
+          x-checker-data:
+            type: git
+            tag-pattern: ^n([\d.]{3,7})$
+
+  # this is used to bundle the final supersonic command correctly and
+  # provides the /app/bin/fyne command to the next module
```

Comment:
so you probably want a cleanup rule `*` to remove it afterwards

Reply:
cleanup rules run at the very end of all the modules? i imagined they ran at the end of the module build...